### PR TITLE
Epmlstrtai 80

### PR DESCRIPTION
**Pull request by roma0297**

### CODE REVIEW

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java [hunk 1]
```diff
@@ -3,11 +3,9 @@
 import com.epam.test_generator.entities.Case;
 import com.epam.test_generator.services.CaseService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.*;
```
Comment (EvilXa0S) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java]: It is considered bad practice to use wildcards in imports. Please import explicitly.

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java [hunk 2]
```diff
@@ -39,10 +37,33 @@ public ModelAndView getCaseById(@PathVariable String caseId) {
...
+                  @RequestParam("numOfSteps") String numberOfStepsInCase, @RequestParam("code") String caseScenarioCode) {
+
+
+        System.out.println(suitId);
+        System.out.println(caseDescription);
+        System.out.println(casePriority);
+        System.out.println(numberOfStepsInCase);
+        System.out.println(caseScenarioCode);
+
+        //Тут, наверное, нужно явно кастить некоторые переменные и делать защиту от тупости пользователя
```
Comment (EvilXa0S) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java]: Let's try to keep comments in English.

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java [hunk 3]
```diff
@@ -39,10 +37,33 @@ public ModelAndView getCaseById(@PathVariable String caseId) {
     }
 
     //It doesn't work
-    @RequestMapping(value = "/suits/{suitId}/cases/{caseId}", method = RequestMethod.PUT)
-    public void updateCaseById(@PathVariable Long caseId, @RequestBody Case caseArg) {
-        System.out.println(caseId + " " + caseArg);
-        casesService.removeCase(caseId);
-        casesService.addCase(caseArg);
+//    @RequestMapping(value = "/saveCase", method = RequestMethod.POST)
```
Comment (EvilXa0S) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java]: I assume this should be discarded and code from target branch should be taken instead. Please evaluate.

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java [hunk 4]
```diff
@@ -26,14 +28,12 @@ public ModelAndView getCasesForSuit(@PathVariable String suitId){
         return model;
     }
 
-    //It doesn't work
```
Comment (pomkine) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java]: Now it works?)

File: Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java [hunk 5]
```diff
@@ -43,11 +43,23 @@ public ModelAndView getCaseById(@PathVariable String caseId) {
...
-    @RequestMapping(value = "/suits/{suitId}/cases/{caseId}", method = RequestMethod.PUT)
-    public void updateCaseById(@PathVariable Long caseId, @RequestBody Case caseArg) {
-        System.out.println(caseId + " " + caseArg);
-        casesService.removeCase(caseId);
-        casesService.addCase(caseArg);
+//    @RequestMapping(value = "/saveCase", method = RequestMethod.POST)
```
Comment (pomkine) [inline Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java]: Can we clean up the commented code here?

File: Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/js/buttonsExecuting.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    var code = $("#code-textarea").val();
+
+
+    if (description === null || description === "" ||
+        priority === null || priority === "") {
+        alert("Fill in all required entry field!");
+        return;
+    }
+
+//            alert(description);
```
Comment (pomkine) [inline Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/js/buttonsExecuting.js]: Do we need these comments?

**Review by Crash1048576 — APPROVED** [review]

**Review by EvilXa0S — APPROVED** [review]

**Pull request merged**